src/main/java/org/lightcouch/View.java

```diff
@@ -438,14 +440,20 @@ public View key(Object... key) {
      *                 keys.
      */
     public View startKey(Object... startKey) {
-        this.startKey = getKeyAsJson(startKey);
-        uriBuilder.query("startkey", this.startKey);
+        this.startKey = getKeyAsJsonElement(startKey);
+        uriBuilder.query("startkey", this.startKey, true);
+        return this;
+    }
+
+    public View startKeyDocId(String startKeyDocId, boolean replace) {
```

Comment:
@mikerhodes @ricellis What cases would we need ```false``` for in ```uriBuilder.query(key, value, false)```?  All the existing test cases and additional complex key test cases are passing with the true value.